### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by sapegin** [styleguidist/react-styleguidist#879]
## To do

(Will be done in #893)

- [x] Tweak colors to comply with WCAG 2 AA contrast ratio (gray text, for example).
- [x] Add an invisible label to the search field (`aria-label`).
- [x] Add [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y)

## To think

- [ ] Missing `lang` attribute in `html` tag. Do we need a config option for that?

Am I missing anything? 

/cc @okonet

**Comment by gergely-nagy** [comment]
I'd like to jump in on this! :rocket:

**Comment by sapegin** [comment]
@gergely-nagy Awesome! 🦄

**Comment by Jlevyd15** [comment]
@sapegin noticed a few color contrast issues with the example Styleguide docs. I've created a PR #1126 to address those. I figured those would land within the scope of this issue but if you think it needs a separate issue let me know.
